plugin-server/src/config/config.ts

```diff
@@ -19,8 +19,10 @@ export function getDefaultConfig(): PluginsServerConfig {
     return {
         INSTRUMENT_THREAD_PERFORMANCE: false,
         OTEL_EXPORTER_OTLP_ENDPOINT: isDevEnv() ? 'http://localhost:4317' : '',
-        OTEL_SDK_DISABLED: isDevEnv() ? true : false,
+        OTEL_SDK_DISABLED: isDevEnv() ? false : true,
```

Comment:
I got this the wrong way round - lucky the helm chart overrides were deployed before hand